nrcan_etl/src/energuide/embedded/region.py

```diff
@@ -1,5 +1,42 @@
 import enum
+import re
 import typing
+import unicodedata
+import Levenshtein
+
+from energuide import logger
+
+
+LOGGER = logger.get_logger(__name__)
+
+
+_FRENCH_NAMES = {
+    'ALBERTA': 'ALBERTA',
+    'COLOMBIE_BRITANNIQUE': 'BRITISH_COLUMBIA',
+    'ILE_DU_PRINCE_EDOUARD': 'PRINCE_EDWARD_ISLAND',
+    'MANITOBA': 'MANITOBA',
+    'NOUVEAU_BRUNSWICK': 'NEW_BRUNSWICK',
+    'NOUVELLE_ECOSSE': 'NOVA_SCOTIA',
+    'NUNAVUT': 'NUNAVUT',
+    'ONTARIO': 'ONTARIO',
+    'QUEBEC': 'QUEBEC',
+    'SASKATCHEWAN': 'SASKATCHEWAN',
+    'TERRE_NEUVE_ET_LABRADOR': 'NEWFOUNDLAND_AND_LABRADOR',
+    'TERRITOIRES_DU_NORD_OUEST': 'NORTHWEST_TERRITORIES',
+    'YUKON': 'YUKON'
+}
+
+_ALTERNATIVE_NAMES = {
+    'YUKON_TERRITORY': 'YUKON',
+    'NORTHWEST_TERRITORY': 'NORTHWEST_TERRITORIES',
+}
+
+_ALTERNATIVE_CODES = {
+    'PEI': 'PE',
+    'NWT': 'NT'
+}
+
+_MINIMUM_SIMILARITY = 0.6
```

Comment:
Why did you pick this value (just curious)?

Reply:
Completely arbitrary trial and error with what seemed to give me good results.

Reply:
sweet.